Gems/GradientSignal/Code/Source/GradientSampler.cpp

```diff
@@ -73,8 +74,8 @@ namespace GradientSignal
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &GradientSampler::AreTransformSettingsDisabled)
                     ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
 
-                    ->DataElement(0, &GradientSampler::m_enableLevels, "Enable Levels", "")
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &GradientSampler::ChangeNotify)
```

Comment:
question: Just checking we definitely want to remove this `ChangeNotify` event?

Reply:
Good catch, I've added the ChangeNotify event back.

Reply:
The ChangeNotify attribute actually isn't needed for any of the data elements anymore, the property handler is doing the enabling and disabling of each group!

Reply:
Okay nice! 🙂